src/app/home-page/widget/starred-box/starred-box.component.html

```diff
@@ -0,0 +1,192 @@
+        <ng-template ngPluralCase="=1"> <img src="../assets/svg/star-on.svg" alt="star" class="mr-1" /> 1 Organization </ng-template>
+        <ng-template ngPluralCase="other">
+          <img src="../assets/svg/star-on.svg" class="mr-1 site-icons-small" /> {{ totalStarredOrganizations }} Organizations</ng-template
+        >
+      </div>
+    </div>
+  </div>
+  <mat-divider></mat-divider>
+  <mat-card-content class="p-3 h-100">
+    <div *ngIf="events.length !== 0 || isLoading; else noEvents">
+      <h5 class="weight-bold">Recent Activity</h5>
+      <div class="size-small widget-list-items" *ngFor="let event of events" fxLayout="row" fxLayoutAlign="space-between center">
+        <div fxLayout="row">
+          <img src="{{ event?.initiatorUser?.avatarUrl }}" alt="User avatar" />
+          <div [ngSwitch]="event.type">
+            <div *ngSwitchCase="EventType.ADDVERSIONTOENTRY" class="my-3">
+              <div class="truncate-text-2">
+                <strong>{{ event.initiatorUser?.username }} </strong> created the {{ event.version?.referenceType | lowercase }}
+                <strong>{{ event.version?.name }}</strong> in {{ event?.tool || event?.apptool ? 'tool' : 'workflow' }}
+                <a
+                  [routerLink]="
+                    event?.apptool
+                      ? '/tools/' + event?.apptool.full_workflow_path
+                      : event?.tool
+                      ? '/tools/' + event?.tool.tool_path
+                      : '/workflows/' + event?.workflow.full_workflow_path
+                  "
+                  >{{
+                    (event?.tool | entryToDisplayName) || (event?.workflow | entryToDisplayName) || (event?.apptool | entryToDisplayName)
+                  }}</a
+                >
+              </div>
+              <div class="subtitle">{{ 'on ' + (event.dbCreateDate | date: 'medium') }}</div>
+              <mat-divider *ngIf="!last" class="custom-margin"></mat-divider>
+            </div>
+            <div *ngSwitchCase="EventType.CREATECOLLECTION" class="my-3">
+              <div class="truncate-text-2">
+                <strong>{{ event.initiatorUser?.username }}</strong> created the collection
+                <a [routerLink]="'/organizations/' + event.organization?.name + '/collections/' + event.collection?.name"
+                  >{{ event.collection?.name }}
+                </a>
+                in organization
+                <a [routerLink]="'/organizations/' + event.organization?.name">{{ event.organization?.displayName }}</a>
+              </div>
+              <div class="subtitle">{{ event.dbCreateDate | date: 'medium' }}</div>
+            </div>
+            <div *ngSwitchCase="EventType.ADDTOCOLLECTION" class="my-3">
+              <div class="truncate-text-2">
+                <strong>{{ event.initiatorUser.username }} </strong> added the {{ event?.tool || event?.apptool ? 'tool' : 'workflow' }}
```

Comment:
In general, whenever a tag will change the formatting of the enclosed content, it's usually a good idea to avoid whitespace between the content and the opening and closing tags, so that the formatting doesn't spill out into the whitespace.  For example:
```
<strong>xyz</strong>
```
rather than
```
<strong>  xyz  </strong>
```

Reply:
ohh okay noted, i'll change it